tests/unit/test_migration.py

```diff
@@ -1280,3 +1280,346 @@ def test_migration_marks_route_migrated(clean_db, fake_cf_client):
     migration = Migration(route, clean_db, fake_cf_client)
     migration.mark_complete()
     assert route.state == "migrated"
+
+
+def test_migration_migrates_happy_path(
+    clean_db, fake_cf_client, iam_commercial, route53, cloudfront, dns, fake_requests
+):
+    """
+    Migrate should:
+    - enable the migration plan
+    - create a migration instance
+    - wait for the migration instance to be created
+    - call update on the migration instance
+    - wait for the update to complete
+    - mark the old instance migrated
+    - call purge on the old instance
+    - disable the migration plan
+    """
```

Comment:
Thank you for laying out the steps here, that is helpful!  It might also be helpful to just denote which step/statement in the test below corresponds with which step.